share/doc/homebrew/Analytics.md

```diff
@@ -0,0 +1,41 @@
+- The Google Analytics anonymous IP setting is enabled i.e. `1` (https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#aip)
+- The Homebrew application name e.g. `Homebrew` (https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#an)
+- The Homebrew application version e.g. `0.9.9` (https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#av)
+- The Homebrew analytics hit type e.g. `screenview` (https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#t)
+
+Homebrew's analytics records the following different events:
+
+- a `screenview` hit type with the Homebrew command you have run (with arguments stripped) e.g. `brew list` (not `brew list foo`)
+- an `event` hit type with the `install` event category, the Homebrew formula you have requested to install e.g. `ack` as the action and an event label e.g. `Mac OS X 10.11, non-/usr/local, CI=1`
+- an `event` hit type with the `BuildError` event category, the Homebrew formula that failed to install e.g. `wget` as the action and an event label e.g. `Mac OS X 10.11, non-/usr/local, CI=1`
+- an `exception` hit type with the `exception` event category, exception description of the exception name e.g. `FormulaUnavailableError` and whether the exception was fatal e.g. `1`
+
+You can also view all the information that is sent by Homebrew's analytics by setting `HOMEBREW_ANALYTICS_DEBUG=1` in your environment. Please note this will also stop any analytics being sent.
+
+It is impossible for the Homebrew developers to match any particular event to any particular user, even if we had access to the Homebrew analytics user ID (which we do not). An example of the most user-specific information we can see from Google Analytics:
+
+![Aggregate user analytics](images/analytics.png)
+
+It is impossible for Google to match a Homebrew analytics user ID to any other Google Analytics user ID. If Google turned evil the only thing they could do would be to lie about anonymising IP addresses and attempt to match users based on IP addresses.
+
+## When?
+Homebrew's analytics are sent throughout Homebrew's execution to Google Analytics.
+
+## How?
+The code is viewable in https://github.com/Homebrew/brew/blob/master/Library/Homebrew/utils/analytics.rb and https://github.com/Homebrew/brew/blob/master/Library/Homebrew/utils/analytics.sh. They are done in a forked, background process and fail fast to avoid delaying any execution. They will fail immediately and silently if you have no network connection.
+
+## Opting-out
+If after everything you've read you still wish to opt-out of Homebrew's analytics you may set `HOMEBREW_NO_ANALYTICS=1` in your environment and we will never send analytics from your machine again.
```

Comment:
"We will never send analytics from your machine while `HOMEBREW_NO_ANALYTICS=1` is in the environment." is a little truthier :)